Formula/newt.rb

```diff
@@ -20,6 +21,8 @@ class Newt < Formula
   def install
     args = ["--prefix=#{prefix}", "--without-tcl"]
 
+    inreplace "configure", "ls /usr/include/python*/Python.h", "find /System/Library/Frameworks/Python.framework/Versions/ -name 'Python.h'"
```

Comment:
This is not the right approach. The `configure` script takes an argument after `--with-python`: see https://pagure.io/newt/c/4feae867000d6369ee036b71ad0b3ac04561bda7?branch=master

Reply:
Thanks, I wasn't aware of that. However, the commit you refer to is not part of release `0.52.20`. There doesn't seem to be a newer release either:

https://pagure.io/releases/newt/

Reply:
I'm not sure what other possibilities there are..
* Lobby upstream `newt` for a new release
* Add support for building the `newt` formula from the git repo, and then use the new `--with-python` option here. I'm not sure if Homebrew's dependency management system can figure that out in a user-friendly way though?